package.json

```diff
@@ -13,13 +13,13 @@
     "posttest": "npm run lint"
   },
   "devDependencies": {
-    "@npmcli/template-oss": "4.21.4",
+    "@npmcli/template-oss": "4.22.0",
     "tap": "^12.7.0"
   },
   "license": "ISC",
   "repository": {
     "type": "git",
-    "url": "https://github.com/npm/node-semver.git"
+    "url": "git+https://github.com/npm/node-semver.git"
```

Comment:
Can we get this change pulled into a `fix` so it's more prevalent in the changelog?